system/src/Grav/Common/Data/Blueprint.php

```diff
@@ -213,14 +213,10 @@ protected function dynamicData(array &$field, $property, array &$call)
         }
 
         list($o, $f) = preg_split('/::/', $function, 2);
-        if (!$f) {
-            if (function_exists($o)) {
-                $data = call_user_func_array($o, $params);
-            }
-        } else {
-            if (method_exists($o, $f)) {
-                $data = call_user_func_array(array($o, $f), $params);
-            }
+        if (!$f && function_exists($o)) {
```

Comment:
This changes the meaning of `if (!$f)`. Old code does nothing of function doesn't exist where new core tries to call `method_exists()` with empty `$f`.